### DISCUSSION TITLE
Prisma Row Level Security with prisma extensions behaves same as supabase?

### DISCUSSION THREAD

**Discussion by Jarrodsz** [prisma/prisma#17616] [Q&A]
supabase is praised for its build in rls.

does the 

https://github.com/sbking/prisma-client-extensions
examples/row-level-security

provides the same functionality?

I read it does not support transactikn querys which would be performance benefit but im not using transactions at all in my code jist regular query and update methods.

So can we conclude if i implement this rls example i benefit from a secure profuction ready rls implementation for prisma?

or aree there issues eith it still beside non transactional querys?

**Comment by nurul3101** [comment]
Hello @Jarrodsz 👋 

Yes, the Client Extension you are referring to does intend to provide the same Row Level Security.

You can use them if you are not heavily reliant on transactions, but please note that Client Extensions still are in preview, so you should be careful and make sure everything is working as expected in case you use them in production.

**Reply by Jarrodsz** [reply]
I look into it thank you.

What would be perhaps be very helpfull is a example repro with these advanched topics demonstrated.

**Reply by nurul3101** [reply]
@Jarrodsz You are looking for this repository: https://github.com/prisma/prisma-client-extensions 😃

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
